Adding career progression inspiration pages to teaching as a career subcategory (#4237)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/teaching-as-a-career/from-teaching-internship-to-leadership-roles.md

---
title: From teaching internship to leadership roles
title_paragraph: |-

>   <p>In Ben’s second year of teaching, he took on the role of head of physics. Only 3 years later, he was promoted to assistant head of maths.</p>

Reviewer: It's a bit weird to say that **assistant** head of maths would be considered a **promotion** above head of physics - I'm not sure that is what the article actually implies?

```suggestion
  <p>In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and become assistant head of maths.</p>
```

(reword as needed)

>   <p>In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and become assistant head of maths.</p>  

Reviewer: ```suggestion
  <p>In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and became assistant head of maths.</p>  
```

description: |-

>   In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and become assistant head of maths.

Reviewer: ```suggestion
  In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and became assistant head of maths.
```

>   In Ben’s second year of teaching, he took on the role of head of physics. Only 3 years later, he was promoted to assistant head of maths.

Reviewer 2: Do we need to change above in-line with Gemma's comments too?
